packages/desktop-client/e2e/budget.test.js

```diff
@@ -50,4 +50,14 @@ test.describe('Budget', () => {
       balance: expect.any(Number),
     });
   });
+
+  test('clicking on spent amounts opens a transaction page', async () => {
+    let categoryName = await budgetPage.getCategoryNameForRow(1);
+    let accountPage = await budgetPage.clickOnSpentAmountForRow(1);
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏to be consistent with the file (but we should really open a bigger discussion on which style we should use and then enforce it via a lint rule; obviously not part of this PR)

```suggestion
    const categoryName = await budgetPage.getCategoryNameForRow(1);
    const accountPage = await budgetPage.clickOnSpentAmountForRow(1);
```